src/_includes/docs/add-to-app/ios-project/limits-common-deps.md

```diff
@@ -0,0 +1,18 @@
+
+Flutter can't handle [common dependencies with xcframeworks][common].
+If both the host app and the Flutter module's plugin define the
+same pod dependency and you integrate Flutter module using this option,
+errors result.
+These errors include issues like Multiple commands produce
+'CommonDependency.framework'.
+
+To work around this issue, link every plugin source in its `podspec` file
+from the Flutter module to the host app's `Podfile`.
+Do this instead of using the plugins' `xcframework` files that the
+[`flutter build ios-framework` command produces][ios-framework].
+
+This issue contributed to the introduction of the
+`--no-plugins` flag for the `flutter build ios-framework` command.
```

Comment:
If I switched to this tab, we haven't introduced `flutter build ios-framework` yet. I'll need to read this entire section before this workaround makes sense to me. Perhaps we should say something like:

```suggestion
In the instructions below, add the `--no-plugins` flag when using the `flutter build ios-framework` command to skip producing `.xcframework`s for plugins.
```